src/olympia/devhub/tests/test_views_versions.py

```diff
@@ -482,6 +486,11 @@ def test_version_history(self):
             'version-reviewnotes-list', args=[self.addon.id, self.version.id])
         assert doc('.review-history-hide').length == 2
 
+        pending_activity_count = doc('.review-history-pending-count')
+        # Only one, for the latest/deleted version
+        assert pending_activity_count.length == 1
+        assert pending_activity_count.text() == '2'
```

Comment:
This `.length == 1` and the following `.text() == '2'`` reads weird, for a moment I thought, "Wait, we're testing that it's one and then it's two?" - just a note